samples/sensor/mcp9808/sample.yaml

```diff
@@ -1,8 +1,7 @@
 sample:
-  description: TBD
-  name: TBD
+  name: MCP9808 Temperature Sensor
 tests:
   test:
-    build_only: true
-    platform_whitelist: frdm_k64f quark_d2000_crb
```

Comment:
im a littlle confused on why for some tests we get rid of the platform_whitelist, but let it remain for others.

Reply:
see comment above.